build-tools/jnienv-gen/jnienv-gen.csproj

```diff
@@ -20,6 +20,16 @@
     <WarningLevel>4</WarningLevel>
     <Externalconsole>true</Externalconsole>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Gendarme|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
```

Comment:
VS needs to change its default `$(DebugType)` value to `portable`.  `full` is BAD.